### PR TITLE
B2C guest configured bundles endpoint

### PR DESCRIPTION
**Pull request by AnastasiiaKSpryker**
## PR Description
Tests created for POST, PATCH, EDIT methods of guest configured bundles endpoint

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/positive.robot [hunk 1]
```diff
@@ -0,0 +1,294 @@
...
+Test Setup    TestSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+
+### POST ###
+Add_configured_bundle_with_1_slot_1_product_new_cart
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/positive.robot:12]: image

File: tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/negative.robot [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+
+Add_configured_bundle_with_nonexistant_template_uuid
+    [Setup]    I set Headers:   Content-Type=${default_header_content_type}     X-Anonymous-Customer-Unique-Id=${random}
+    When I send a POST request:    /guest-carts//guest-configured-bundles    {"data": {"type": "guest-configured-bundles","attributes": {"quantity": 1,"templateUuid": "not_uuid","items": [{"sku": "${configurable_bundle_slot_1_product_1}","quantity": 1,"slotUuid": "${configurable_bundle_slot_1_uuid}"}]}}}
+    Then Response status code should be:    422
+    And Response reason should be:    Unprocessable Content
+    And Response should return error code:    4002
+    And Response should return error message:    Configurable bundle template not found.
+
+Add_configured_bundle_with_empty_slot_uuid
```
Comment (OKrivtsova) [inline tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/negative.robot:56]: tests fail
image

File: tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/positive.robot (hunk 1 above)
Comment (AnastasiiaKSpryker) [reply tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/positive.robot:12]: Seems I had wrong availability data locally, updated product SKU to available one - now all tests are passing on stage

File: tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/negative.robot (hunk 2 above)
Comment (AnastasiiaKSpryker) [reply tests/api/b2c/glue/configured_bundles_endpoints/guest_configured_bundles/negative.robot:56]: Same as above - updating product SKU in test data file helped

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**